AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockImageDetailsDeleteActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAdobeStockImageDetailsDeleteActionGroup" extends="AdminEnhancedMediaGalleryImageDetailsDeleteActionGroup">
+        <annotations>
+            <description>Delete image from the View Details panel</description>
+        </annotations>
+
+        <wait time="3" stepKey="waitForLoadingMask" before="waitForLoadingMaskToDisappear"/>
```

Comment:
@engcom-Foxtrot is this change specific to Adobe Stock Integration? Do you think it will make sense to update the `AdminEnhancedMediaGalleryImageDetailsDeleteActionGroup` in magento2 repo instead?

Reply:
@sivaschenko yes, it's the Adobe Stock Integration specific behavior.